README.md

```diff
@@ -19,14 +19,49 @@ The default server directory is the current working directory.
 
 ### Programatic Usage
 
-You can use IPX as a Connect/Express middleware or directly use ipx api.
+You can use IPX as a middleware or directly use ipx api.
+
+#### H3 Middleware
+[H3](https://github.com/unjs/h3) is a minimal h(ttp) framework built for high performance and portability. It is part of the [unjs](http://unjs.io) ecosystem to which `ipx` belongs.
 
 ```js
+import { createApp, eventHandler, toNodeListener } from "h3";
+import { createIPX, createIPXMiddleware } from "ipx";
+import { listen } from "listhen";
+
+const app = createApp();
+const ipx = createIPX({
+  domains: ['unjs.io']
+});
+
+const middleware = createIPXMiddleware(ipx)
+
+const handler = eventHandler(async (event) => {
+  await middleware(event.node.req, event.node.res);
+});
+
+app.use("/image", handler);
+
+listen(toNodeListener(app));
```

Comment:
Not 100% sure the `toNodeListener` is required here

Reply:
It is necessary. Without this function, the server launches but each endpoint loads endlessly in the browser and does not start.

This is also in the README h3 example: https://github.com/unjs/h3#usage

You can also make a clone from the template: https://github.com/oritwoen/ipx-h3 and take a look yourself.